### ISSUE TITLE
Skip button disappeared form quick labeling

### ISSUE THREAD

**Issue by alex-eurec** [heartexlabs/label-studio#1167]
**Describe the bug**
Skip button disappeared form quick labeling:

**To Reproduce**
1. Install LS v1.1.0
2. Create user, create project and upload 1 or more text(s)
3. Select one to enter quick labeling (do not select "Label all tasks" because there, it is working OK)
![image](https://user-images.githubusercontent.com/81571132/125083676-f85f2c80-e0c8-11eb-9988-5fdf93c20e96.png)

**Expected behavior**
The skip button to be visible and the shortcut `ctrl`+`space` working

**Screenshots**
Before
![image](https://user-images.githubusercontent.com/81571132/125083634-ec736a80-e0c8-11eb-9267-ce67163df16e.png)

After
![image](https://user-images.githubusercontent.com/81571132/125083676-f85f2c80-e0c8-11eb-9988-5fdf93c20e96.png)

**Environment:**
 - OS: Ubuntu Desktop 18.04
 - Label Studio Version 1.1.0
 - pip installation inside Venv with only python3.6.9

**Comment by makseq** [comment]
It's expected behaviour, skip button is available only in Labeling  mode (via blue Label button).

**Comment by alex-eurec** [comment]
We (my team and I) found easier to use it in the quick label mode.
Nevertheless, as this is an expected change, I am going to close the issue

**Comment by makseq** [comment]
@alex-eurec What is your workflow of a skip button usage?

**Comment by alex-eurec** [comment]
We prefilter some texts (from hundred to hundred thousands), and then click on one of the text results.
Then, we enter in the "Quick label mode" and we start labeling. If we want to skip to the next text because the present one is not interesting, we either use the shortcut or press "Skip".

We normally use "quick label mode" instead of clicking the "Label all tasks" because in the magnitude of hundred thousand texts to categorize, it last too long to charge the texts, "quick labeling mode" is almost instant.

**Comment by makseq** [comment]
@alex-eurec do you mean you have a big delay between the current task and the next in the Label stream?

**Comment by alex-eurec** [comment]
Yes, and the delay is also at the beginning of the labeling itself, after the "Label all tasks" button is pressed.

For example, with 88753 task to label, it takes about 3.12 minutes in this screen with the moving dots. (I timed it)
![image](https://user-images.githubusercontent.com/81571132/125903051-1f3f8572-dbcc-41cd-8d12-3bd9cacf7c78.png)

After that, the time between current and next task is about 1.10 minutes (same image as the one above)

**Extra information**
- I have deactivated the "Retrieve predictions when loading a task automatically" option.
- I can see that LabelStudio process is working 100% but just in one core. ¿It is prepared for multi-core?
    ![image](https://user-images.githubusercontent.com/81571132/125904179-54d28216-0603-4331-b4d3-26b26cc8eb03.png)
- Total amount of tasks in the example project: 99999
- The resources on the machine running LabelStudio seems to be OK (16GB RAM)
    ![image](https://user-images.githubusercontent.com/81571132/125904510-ed079ab1-efe0-4fe0-99e7-5c96887e2525.png)



Note: the dark skin is just "night eye" doing its job.
